### PR TITLE
Add /locale command

### PR DESCRIPTION
**Pull request by kevinlul**
For #47

### CODE REVIEW

**Comment by kevinlul** [comment]
Further work I could see in the scope of this PR are reducing the nesting in the command implementation.

File: docs/commands/locale.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Check or set Bastion's locale for this channel or server. _This command is in development._
+
+This setting is meant to apply to all text displayed by the new instance of Bastion, but
+translations may not be complete, in which case it will fall back to English. Features
+still handled by the old instance will not be affected.
+
+## Definitions
+
+**Locale**: a parameter affecting _localization_, such as what language and dialect are used,
```
Comment (AlphaKretin) [inline docs/commands/locale.md:11]: are we using -ization or -isation? I'm used to -isation and I thought we were using en-AU, but en-AU isn't strictly en-GB and I could believe -ization is technically correct.

**Review by AlphaKretin — APPROVED** [review]
Barring the one potentially-wrong spelling thing in the docs, this looks good to me

File: docs/commands/locale.md (hunk 1 above)
Comment (kevinlul) [reply docs/commands/locale.md:11]: Oops it should be _isation_ for both, it's Canadian follows the American spelling for this one.

**Pull request merged**